keyboards/chew/info.json

```
@@ -0,0 +1,87 @@
+{
+    "manufacturer": "florent@linguenheld.fr",
+    "keyboard_name": "chew",
+    "maintainer": "florent@linguenheld.fr",
+    "bootloader": "rp2040",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "direct": [
+            ["GP4", "GP3", "GP2", "GP1", "GP0"],
+            ["GP15", "GP26", "GP27", "GP28", "GP29"],
+            ["GP14", "GP13", "GP9", "GP8", "NO_PIN"],
+            ["GP7", "GP6", "GP5", "NO_PIN", "NO_PIN"]
+        ]
+    },
+    "processor": "RP2040",
+    "split": {
+        "enabled": true,
+        "matrix_pins": {
+            "right": {
+                "direct": [
+                    ["GP0", "GP1", "GP2", "GP3", "GP4"],
+                    ["GP29", "GP28", "GP27", "GP26", "GP15"],
+                    ["GP8", "GP9", "GP13", "GP14", "NO_PIN"],
+                    ["GP5", "GP6", "GP7", "NO_PIN", "NO_PIN"]
+                ]
+            }
+        },
+        "transport": {
+            "protocol": "serial_usart",
+            "sync": {
+                "matrix_state": true
+            }
```

Syncing matrix state with no rgb/oled is somewhat pointless.

```suggestion
            "watchdog": true
```